qiskit/utils/__init__.py

```diff
@@ -18,34 +18,27 @@
 
 .. currentmodule:: qiskit.utils
 
-.. autosummary::
-   :toctree: ../stubs/
-
-   add_deprecation_to_docstring
-   deprecate_arg
-   deprecate_arguments
-   deprecate_func
-   deprecate_function
-   local_hardware_info
-   is_main_process
-   apply_prefix
-   detach_prefix
-   wrap_method
+.. autofunction:: add_deprecation_to_docstring
+.. autofunction:: deprecate_arg
+.. autofunction:: deprecate_arguments
+.. autofunction:: deprecate_func
+.. autofunction:: deprecate_function
+.. autofunction:: local_hardware_info
+.. autofunction:: is_main_process
+.. autofunction:: apply_prefix
+.. autofunction:: detach_prefix
+.. autofunction:: wrap_method
 
 Algorithm Utilities
 ===================
 
-.. autosummary::
-   :toctree: ../stubs/
-   :nosignatures:
-
-   summarize_circuits
-   get_entangler_map
-   validate_entangler_map
-   has_ibmq
-   has_aer
-   name_args
-   algorithm_globals
+.. autofunction:: summarize_circuits
+.. autofunction:: get_entangler_map
+.. autofunction:: validate_entangler_map
+.. autofunction:: has_ibmq
+.. autofunction:: has_aer
+.. autofunction:: name_args
+.. autodata:: algorithm_globals
```

Comment:
Huh, I either hadn't had any luck with `autodata`, or for some reason I thought it didn't exist.  Good to see it both in use and working here.